examples/kft-v2/scripts/trl_training.py

```
@@ -0,0 +1,790 @@
+        os.environ['MASTER_ADDR'] = master_addr
+        os.environ['MASTER_PORT'] = master_port
+
+        if world_size > 1:
+            try:
+                torch.distributed.init_process_group(
+                    backend='gloo',
+                    rank=global_rank,
+                    world_size=world_size
+                )
+                torch.distributed.barrier()
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)